### ISSUE TITLE
Log viewer resets DataTable/Graph split when window is made wider

### ISSUE THREAD

**Issue by rmackay9** [ArduPilot/MissionPlanner#2333]
Making the log viewer window winder unnecessarily affects the position of the split between the graphing area (at the top) and the data table area (at the bottom).

Here is a picture of the window before:
![image](https://user-images.githubusercontent.com/1498098/74495865-501d5680-4f1c-11ea-8d2e-18eaba895699.png)

And here it is a moment later after I have grabbed the right side and made it winder by a few pixels
![image](https://user-images.githubusercontent.com/1498098/74495879-5c091880-4f1c-11ea-9c7c-59a3260bd60d.png)

**Comment by meee1** [comment]
https://github.com/ArduPilot/MissionPlanner/blob/master/Log/LogBrowse.cs#L3165
atm when you resize the window it does a 50% split

**Comment by rmackay9** [comment]
@meee1, yup, just not quite sure why it does that.  From a user point of view, making it wider shouldn't affect the vertical split.  Ideally it should maintain the current balance between up and down as the window is made wider or taller.
